src/box/space.c

```diff
@@ -1022,6 +1025,8 @@ space_run_replace_triggers(struct event *event, struct txn *txn,
 int
 space_on_replace(struct space *space, struct txn *txn)
 {
+	if (!space_event_is_enabled)
+		return trigger_run(&space->on_replace, txn);
```

Comment:
Better put it in `space_run_recovery_triggers`? Don't like branching with duplicate code.